### ISSUE TITLE
[Group Request] Create a new chapter "OpenTelemetry (OTel) Berlin"

### ISSUE THREAD

**Issue by jpkrohling** [cncf/communitygroups#150]
### City or location name for your CNCG

Berlin

### What country do you want to start your community group in?

Germany

### Tell us about yourself. List your name, email, company, GitHub URL, LinkedIn URL, and CNCF Slack workspace name (or if you need to be invited), and a required profile link on community.cncf.io

Juraci Paixão Kröhling, juraci.kroehling@grafana.com , Grafana Labs, https://github.com/jpkrohling, https://www.linkedin.com/in/jpkroehling/ , "Juraci Paixão Kröhling", https://community.cncf.io/u/mmzqmh/#/about

### What is your mission statement or objective for this new chapter?

Welcome to the Berlin OpenTelemetry (OTel) Meetup Group! This is the perfect place for anyone interested in observability to come together to learn, share knowledge, and network with like-minded individuals. Whether you're just starting out with OTel or are a seasoned pro, this group is for you. Join us for regular meetups where we'll discuss best practices, showcase demos, and dive into the latest trends in observability. Let's explore how OpenTelemetry can help us gain valuable insights into our systems and improve overall performance. We can't wait to connect with you!

### Code of Conduct

- [X] I/We agree to follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md)

### Where organizers reside

- [X] I/We confirm I/we am/are local to or live near the city of the above-proposed chapter.

### Organizers companies/affiliations

- [X] I/We agree if there is more than one organizer now or onboarded in the future, the majority come from different companies

### Inclusive

- [X] I/We agree. To assure this, I/we have recently completed the [Inclusive Open Source Community Orientation (LFC102) course](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)

### Vendor Neutral

- [X] I/We have read what [vendor-nuetrality means](https://contribute.cncf.io/maintainers/community/vendor-neutrality/), and agree that the main content of our meetups will remain vendor-nuetral

### Agree to use CNCG branding

- [X] I/We agree to use the CNCG logo in my/our chapter and social branding

### Registration through CNCF platforms

- [X] I/We agree to host free registration in the main CNCF platform

### Remain an active chapter

- [X] I/We agree to have a meetup at least once every 90 days, or else communicate reasoning for upcoming inactivity.

### Collaboration with other chapters

- [X] I/We acknowledge the CNCF right to approve additional chapters in my area based on this criteria

### Aim to have a diverse representation of speakers in your meetups

- [X] I/We will do our best to host diverse speakers throughout the year

**Comment by jpkrohling** [comment]
I'm actively looking for co-organizers for this one.

**Comment by clemra** [comment]
Hi Juraci, as discussed, we'd be glad to help with organizing this. 

Clemens Rawert, [clemens@langfuse.com](mailto:clemens@Langfuse.com) , [Langfuse](https://langfuse.com), https://github.com/clemra, https://linkedin.com/in/rawert

**Comment by jpkrohling** [comment]
To circle back: I've spoken to a few other CNCF Ambassadors and there are some concerns about this clashing with CNCF Berlin, a group that exists but hasn't had an event for over a year. I do believe that we could have an OTel dedicated charter, but I'm not ready to have this discussion right now: I have quite a few things on my plate right now and I'd rather see "some" event happening than figuring out the perfect solution.

Therefore, I asked our Grafana Labs' folks to create a group for this initial event, and we'll see how it goes from there: https://www.meetup.com/otel-berlin/

In summary, please do not create this charter just now; let's see how it goes after this initial event.

**Comment by clemra** [comment]
Thank you, Juraci. I agree, let's see how this resonates first. I joined the group on Meetup.

**Comment by jpkrohling** [comment]
We had our first event yesterday using an external meetup group, and I believe it was a success: about 20 people joined from the ~50 that RSVPed, plus another ~50 were on the waiting list. Right now, the meetup group has 150 people.

I'd be open to co-organizing the next one with @Sabariesh, and I believe we have a location for it already. My initial plan is to have it on Tuesday, Oct 15.

As such, I would like to move forward with this new charter, as I still believe there's appetite for an OTel-only group.

**Comment by AudMonte01** [comment]
Thank you for submitting this tissue @jpkrohling.

The best way to go about this is to be added as a co-organizer to the current Berlin chapter, which could use some organizational diversity (right now it's just Microsoft) and more activity (last Meetup was in June and we have a 90-day activity standard.

If this is of interest, I will put this in front of those organizers to see if they can at least welcome you as an organizer and you can focus on the OTel meetups within the cloud native group.

Otherwise, since this is project-focused, then you can be better serves through the projects teams service desk. Cc @k8tgreenley

**Comment by jpkrohling** [comment]
For status: yes, there's interest in joining any existing CNCF meetup at the moment, or to have a new charter dedicated to OTel. The only thing I ask is that we can have our OTel meetup independently from other events: there's enough audience in Berlin for a dedicated OTel group, sharing more advanced experiences for practitioners, as evidenced by our September and November events.

**Comment by AudMonte01** [comment]
Hello @jpkrohling - I have added you as an organizer to the Berlin chapter. You should have received an automated welcome email from me. I have also added you to the global organizer Slack channel in the CNCF Workspace.

Please add a profile image of you in your community.cncf.io profile so we can make you as an organizer forward-facing on the chapter page.

This ticket is now closed.

**Comment by jpkrohling** [comment]
Thank you!
